CONTRIBUTING.md

```diff
@@ -86,6 +87,51 @@ assessment.
 
 ## Discussing Mozilla's Position on a Web Specification
 
+### For Mozilla's subject-matter experts
```

Comment:
I would move this below the "broader community" section.

Reply:
Yeah, I initially wanted to do that except that I felt that "broader community" needed to be relative to some antecedent.  I guess I can just do it anyway.